snowfall/training/test_compute_expected_times.py

```diff
@@ -0,0 +1,97 @@
+#!/usr/bin/env python3
```

Comment:
@danpovey 
Here is the test script. I cannot find any problems in the script.

According to the output, there are no repeated phones in the phone_seqs associated with the n-best paths for both `mbr_lats` and `den_lats`.

We can see from the output that  a significant port of entries in the phone_seqs from`mbr_lats`  are 0s.